Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py

```diff
@@ -344,6 +348,33 @@ def get_url_indicator_reputation(
     )
 
 
+def get_email_indicator_reputation(stix_object: dict, reputation_score: int, reliability_score: str, tlp: str) -> CommandResults:
+    """
+    Return stix_object of type email as indicator
+    """
+
+    dbot_score = Common.DBotScore(
+        indicator=stix_object["name"],
+        indicator_type=DBotScoreType.EMAIL,
+        integration_name=INTEGRATION_NAME,
+        score=reputation_score,
+        reliability=reliability_score,
+    )
+
+    email_addr = extract_indicator_from_pattern(stix_object["pattern"])
+    if not email_addr:
+        return
```

Comment:
There's a problem with mypy here because it expects a return value since you defined CommandResults in the function signature

Reply:
\o/ :facepalm: